Resources/public/css/theme/views/loginform.css

```diff
@@ -24,8 +24,9 @@
 }
 
 .ez-view-loginformview .ez-loginform-title {
-    font-size: 0.9rem;
+    font-size: 0.95rem;
     color: #444;
+    font-family: sans-serif;
```

Comment:
Ok, changed back in b22daf33facb923f1f9792aec29d48f851f3b669, thanks for input/review :)